now/cli/__init__.py

```diff
@@ -7,12 +7,13 @@
 
 import cpuinfo
 
+from now import __version__ as version
 from now.deployment.deployment import cmd
 
 warnings.filterwarnings("ignore")
 
 if len(sys.argv) != 1 and not ('-h' in sys.argv[1:] or '--help' in sys.argv[1:]):
-    print('Initialising Jina NOW...')
+    print(f'Initialising Jina NOW {version}...')
```

Comment:
adding `v` before version would probably be more clear for user? `v0.0.21`
```suggestion
    print(f'Initialising Jina NOW v{version} ...')
```